docs/04-guides/playback-a-video-stream.mdx

```diff
@@ -6,16 +6,166 @@ description: Playback video and livestreams
 import CodeBlock from "@theme/CodeBlock";
 import CodeBlockTabs from "@site/src/components/CodeBlockTabs";
 import CodeBlockTabItem from "@site/src/components/CodeBlockTabItem";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Player from "@site/src/components/Player/Livepeer";
+import HTMLPlayer from "@site/src/components/Player/HTML";
 
 # Playback a video
 
-To playback your live or on demand video inside your application you'll need to
-use a video player component that supports the HLS protocol. While there are
-many HLS video players out there, we recommend using
-[livepeer.js](https://livepeerjs.org).
+This guide provides you the instructions to playback a video or a live stream on
+your website/applications. To playback live streams or on-demand video inside
+your application, you'll need to use a video player component that supports the
+HLS protocol.
 
-## Audio-only streams
+You can either use Livepeer's built-in player or your own player. This guide
+covers both options.
 
-If you want to provide only the audio version of the stream, you can use
-?video=false at the end of an .m3u8 URL to get an audio-only stream. For
-example, `https://livepeercdn.com/hls/{playbackId}/index.m3u8?video=none`
+## Using Livepeer's player
```

Comment:
Should we use a tabbed display, like the tutorials for API / dashboard? One for livepeer.js and one for "custom player".

Then we should later add one bringing back the "embeddable player" (lvpr.tv) docs as well, which is still the easiest way to integrate if you're not using React. Don't worry about this one now tho, we had docs for this in the old repo but they got removed on this `studio-docs` one. IMO we should bring them back, but it isn't a regression of this PR anyway.